recipes/pywinpty/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pywinpty" %}
+{% set version = "0.1.0" %}
+{% set hash = "106b9fcf793474626c61bb792e8ce1d850dc383081059f81bbf3028ff8abcb4c" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/spyder-ide/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 0
+  skip: True  # [osx or linux or py27]
+  features:
+    - vc14  # [py35 or py36]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - pywin32
+    - winpty
+    - vc 14  # [py35 or py36]
```

Comment:
@jakirkham, not really. See https://github.com/conda-forge/python-feedstock/issues/104